css/style.css

```diff
@@ -0,0 +1,1248 @@
+  visibility: hidden;
+}
+
+@media screen and (min-width: 50em) {
+
+  .portfolio-slider-caption {
+    top: 20%;
+    right: 25%;
+  }
+
+  .portfolio-slider-bullets[aria-hidden="true"] {
+    visibility: visible;
+  }
+
+  .portfolio-slider-bullets {
+    padding: .3em;
+    font-size: 1.777em;
+    text-align: center;
+  }
+
+  .portfolio-slider-bullets li {
+    display: inline;
+  }
+
+  [title$="bullet"]:before {
+    content: "•";
+    color: #d4d4d4;
+  }
+
+  .slider-bullet-active:before {
+    color: #fd6632;
+  }
+}
+
+
+
+/*
+PORTFOLIO NAVIGATION
+*/
+
+[title="portfolio-categories"] li {
+  display: block;
+  margin-top: .9em;
+  margin-bottom: .5em;
+  text-align: center;
+}
+
+.portfolio-menu-item {
+  padding: .3em;
+  font-size: 1.333rem;
+  text-decoration: none;
+  color: #d4d4d4;
+}
+
+.portfolio-selected-category {
+  color: #fff;
+  background-color: #fd6632;
+}
+
+@media screen and (min-width: 50em) {
+
+
+  [title="portfolio navigation"] {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-basis: 100%;
+    flex-basis: 100%;
+    -webkit-justify-content: center;
+    justify-content: center;
+    margin-top: 1em;
+    margin-bottom: 1.777em;
+  }
+
+  [title="portfolio-categories"] li {
+    display: inline-block;
+  }
+
+  .portfolio-menu-item {
+    font-size: 1rem;
+  }
+}
+
+
+/*
+PORTFOLIO ITEMS
+*/
+
+.portfolio-item {
+  position: relative;
+  max-width: 300px;
+  margin-top: .5em;
+  margin-bottom: 2em;
+  text-align: center;
+}
+
+.portfolio-opacity {
+  opacity: .7;
+}
+
+.portfolio-link-hover:hover {
+  display: block;
+  background: #000;
+}
+
+.portfolio-link-hover:hover:after {
+  position: absolute;
+  top: 30%;
+  left: 42%;
+  width: 1.777em;
+  height: 1.777em;
+  padding-top: .4em;
+  content: "\e608";
+  font-family: 'ico-fonts';
+  font-size: 1.6em;
+  text-align: center;
+  color: #fff;
+  background-color: #fd6632;
+  border-radius: 50%;
+}
+
+.portfolio-caption-header {
+  padding: .5em 0;
+  font-weight: bold;
+  font-size: 1.333em;
+}
+
+.portfolio-category {
+  color: #7c8387;
+}
+
+/*
+ABOUT
+*/
+
+.about, .about p {
+  color: #b9bdbf;
+}
+
+.about {
+  background-color: #252629;
+  padding: 4em 1em 1em;
+}
+
+.about-header, .skills-header {
+  color: #fff;
+  text-decoration: underline;
+}
+
+.about-header {
+  font-size: 2.4em;
+  margin-bottom: 0.5em;
+}
+
+.about p {
+  display: inline-block;
+  line-height: 1.5em;
+  margin-bottom: 1em;
+}
+
+.skills-header {
+  font-size: 1.7em;
+  margin-bottom: 0.8em;
+}
+
+.skills li {
+  margin-bottom: 1em;
+}
+
+.progress-header {
+  text-align: left;
+  margin-bottom: 0.3em;
+}
+
+progress[value]::-webkit-progress-bar {
+  background-color: #3a3a3a;
+}
+
+progress[value]::-webkit-progress-value {
+  background-color: #fd6632;
+}
+
+.join-us-header {
+  font-weight: normal;
+  font-size: 1.5em;
+  margin-bottom: 1em;
+}
+
+.contact-us-button {
+  font-weight: bold;
+  padding: 0.9em 3.5em;
+}
+
+.about>aside {
```

Comment:
Be consistent in your spacing. You have `.about p` earlier, and now `.about>aside`. Best practice is to apply spaces around operators and between elements: `.about > aside`.